### PR TITLE
Fix editor crashing while using composed characters on Text Element

### LINKED ISSUE #1836: Editor crashing when writing in Japanese
## Bug Description

I wrote some text in Japanese, the editor crashed. :'( 

## Expected Behaviour

Don't want it to crash. 

## Steps to Reproduce

See video. 

## Screenshots

[Editor crashing when writing JP text.zip](https://github.com/google/web-stories-wp/files/4647437/Editor.crashing.when.writing.JP.text.zip)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## QA Instructions
1. Add a text field
1. Click to enter edit mode
1. Use the Mac OS X built-in Hiragana keyboard
1. Type the characters "kon" and press <kbd>space</kbd>
1. Observe that Japanese characters are correctly inserted rather than the whole editor crashing.

### PR DESCRIPTION
**Pull request by obetomuniz**
## Summary

It only fixes editor crashing when using text elements with composed characters. Additional bugs related to the usage of composed characters will be fixed in follow-up PRs (e.g. #2002)

## Relevant Technical Choices

- This PR only avoids consuming `getStyle` from a node that not exists.
- The test added only ensure that we can render normally composed characters

## User-facing changes

It will avoid breaking the editor. It also allows use composed characters in some ways (pressing <kbd>enter</kbd> or a non-composed character before leaving the edit mode on the text element), but there are still issues to improve the usage of composed characters.

## Testing Instructions

- Change your computer to Japanese (Hiragana) [MacOS process]
- On Web Stories Editor, add a new Text Element
- Enter on Edit Mode of the Text Element added previously and type `konnichi` in your QWERTY keyboard, which is expected to render こんにち on the text element editable area.
- Press <kbd>enter</kbd>
- Click outside to leave the Edit Mode of the Text Element

---

Solving #1836

### CODE REVIEW

**Review by barklund — COMMENTED** [review]
Just wanted to check - do any of these tests work as regression tests? Do they fail without the fix in place? If not, they're not really adding anything. But if they do, awesome!

**Comment by obetomuniz** [comment]
@barklund they only ensure that we can render composed characters without problems in the editor, which is directly related to the original issue. Correct if I am wrong, but as discussed today on [Slack](https://xwp.slack.com/archives/C0135V8LYJE/p1590594360091200), we are not interested in test OS stuff, which is the another aspect of the given issue and I am not testing here.

File: assets/src/edit-story/elements/text/karma/edit.karma.js [hunk 1]
```diff
@@ -122,6 +122,39 @@ describe('TextEdit integration', () => {
...
+
+        await fixture.events.keyboard.type('こんにち');
+
+        await fixture.snapshot('after content changes');
+
+        // Exit edit mode by clicking right outside the editor.
+        // @todo: this API is a bit low level. would be good to add additional
+        // ways to do events relative to targets and each other. Maybe something
+        // like `move({relative: element, anchor: 'topleft', x: -10})`?
+        const { x, y } = editor.getBoundingClientRect();
```
Comment (dvoytenko) [inline assets/src/edit-story/elements/text/karma/edit.karma.js]: Let's hoist this into a function. E.g. `exitEditor(editor)`.

**Comment by barklund** [comment]
> @barklund they only ensure that we can render composed characters without problems in the editor, which is directly related to the original issue.

My question was, if the test fails without the fix. And it doesn't fail. If I revert the code change in `draftUtils` but keep the tests, all tests pass.

Thus the tests don't actually serve as a regression towards finding this bug and verifying it is fixed. If someone later changes something in `draftUtils` that would make the same bug re-appear, the available tests would not catch it.

I don't know how we would catch it though, without going in to all sorts of emulated input methods, but these tests - as they are right now - really do nothing new and thus doesn't add anything. IMHO.

**Comment by obetomuniz** [comment]
@barklund Yeah, I can see your point perfectly. In the other hand, the tests here are only _smoke_ tests given the nature of the bug. That's why they are not adding new areas of coverage, but only enhancing in some way what we already have, only granting that we support composed characters as we have to support. Anyway, I will remove them for further investigation in a follow up PR (e.g. related to #2002), so we can move forward this PR-fix. Thoughts? Also, thanks for share of your concerns on it. I also agree, its a little bit grey what we should do here.

**Review by barklund — APPROVED** [review]
I can approve of this change without a test, as we aren't really equipped to properly test this yet.

**Pull request merged**